avocado/core/safeloader.py

```diff
@@ -630,9 +630,9 @@ def find_python_tests(module_name, class_name, determine_match, path):
 
             modules_paths = [parent_path,
                              os.path.dirname(module.path)] + sys.path
-            try:
-                _, found_ppath, _ = imp.find_module(parent_module, modules_paths)
-            except ImportError:
+            found_ppath = importlib.machinery.PathFinder.find_spec(
+                parent_module, modules_paths).origin
+            if found_ppath is None:
```

Comment:
Hi @tiagohonorato, I don't understand the purpose of this If statements.

Reply:
Since the try.. except was removed, I guess it is used as a replacement, to continue or not.

Reply:
Yes, of course :man_facepalming: thanks for explanation.

Reply:
Hi @richtja and @beraldoleal, thanks for the review! Adding on @beraldoleal's comment, the `find_spec` method does not raise `ImportError`, instead it returns `None`. So, that's why I replaced the try... except with the if statement.